src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,47 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
+    public static final int MAX_MEMORY = 3;
+
     public static void main(String[] args) {
 
+        Scanner sc = new Scanner(System.in);
+
+        Memory[] memories = new Memory[MAX_MEMORY];
+
+        int memorycount = 0;
+
+        System.out.println("Podaj informację o trzech unikalnych pamięciach RAM");
+
+        while (memorycount < memories.length) {
+            System.out.println("Podaj rodzaj pamięci");
+            String name = sc.nextLine();
+            System.out.println("Podaj pojemność pamięci");
+            int size = sc.nextInt();
+            sc.nextLine();
+            Memory tempMemory = new Memory(name, size);
```

Comment:
To możesz wydzielić do osobnej metody
```
Memory tempMemory = createMemory();
```
Możesz wtedy przenieść do niej Scannera, bo będzie używany tylko tam